packages/fether-react/src/Send/Send.js

```diff
@@ -5,18 +5,25 @@
 
 import React, { Component } from 'react';
 import { Route, Redirect, Switch } from 'react-router-dom';
+import { provideTokens } from '../contexts/TokensContext.js';
+import withAccount from '../utils/withAccount.js';
 
 import Sent from './Sent';
 import Signer from './Signer';
 import TxForm from './TxForm';
 
+@withAccount
+@provideTokens
 class Send extends Component {
   render () {
+    const {
+      match: { path }
+    } = this.props;
     return (
       <Switch>
-        <Route exact path='/send/:tokenAddress' component={TxForm} />
-        <Route path='/send/:tokenAddress/signer' component={Signer} />
-        <Route path='/send/:tokenAddress/sent' component={Sent} />
+        <Route exact path={`${path}`} component={TxForm} />
+        <Route path={`${path}/signer`} component={Signer} />
+        <Route path={`${path}/sent`} component={Sent} />
```

Comment:
We can put `<Route path='/send:tokenAddress/from/:accountAddress/signer' component={Signer} />` here.